app/utils/user/index.ts

```diff
@@ -285,3 +285,27 @@ export function filterProfilesMatchingTerm(users: UserProfile[], term: string):
             some((suggestion) => suggestion.startsWith(trimmedTerm));
     });
 }
+
+export function getNotificationProps(user: UserModel) {
+    if (user && user.notifyProps) {
+        return user.notifyProps;
+    }
+
+    const props: Partial<UserNotifyProps> = {
+        channel: 'true',
+        comments: 'any',
+        desktop: 'all',
+        desktop_sound: 'true',
+        email: 'true',
+        first_name: (!user || !user.firstName) ? 'false' : 'true',
+        mention_keys: user ? `${user.username},@${user.username}` : '',
+        push: 'mention',
+        push_status: 'online',
+
+        //todo: verify if email_threads and push_threads do exist
```

Comment:
Ticket needed?

Reply:
I've looked up those 2 values on the 'User' section on the API docs and I couldn't find them over there; which led me to think that they might be obsolete values that require cleanup.  I'll be removing them and in case needed, I'll add them back.